gulpfile.js

```diff
@@ -0,0 +1,66 @@
+/**
+@license
+Copyright (c) 2018 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+*/
+
+const gulp = require('gulp');
+const del = require('del');
+const { exec } = require('child_process');
+const { renameSync: move } = require('fs');
+const renamer = require('renamer');
+const replace = require('replace');
+
+/**
+ * Cleans the PRPL server build directory (<project folder>/server/build)
+ */
+gulp.task('clean:prpl-server', () =>
+  del`server/build`);
+
+/**
+ * Builds a static version of the PWA that can be used on any hosting service
+ */
+gulp.task('build:static', (cb) =>
+  // Build the project using Polymer CLI
+  exec('polymer build', cb));
+
```

Comment:
Would you mind removing this unused gulp task too? Thanks!

Reply:
Sorry, forgot about that one!